lib/tic_tac_toe_checker/application.ex

```diff
@@ -27,10 +24,14 @@ defmodule TicTacToeChecker.Application do
 
   @impl Application
   def prep_stop(state) do
-    IO.inspect(state)
+    state
+    |> inspect()
+    |> IO.puts()
```

Comment:
How is this better? :)

Reply:
Credo doesnt respect IO.inspect, but okay with IO.puts)

Reply:
> Credo doesnt respect IO.inspect, but okay with IO.puts)

🤣 

▸ [BAD] suppress a warning https://hexdocs.pm/credo/config_comments.html
▸ [GOOD] use `Logger.info/1`